### PR TITLE
Add label and messages to textarea inputs

### PR DESCRIPTION
**Pull request by JammingBen**
Similar to https://github.com/owncloud/owncloud-design-system/pull/1141:

* Remove placeholders
* Add required label to textareas
* Add optional description message to textareas
* Add aria-describedby which either references the description (if given), or an error/warning (if given)
* Add aria-invalid property

Unfortunately there was no message line implemented yet, so I had to include this first.

### CODE REVIEW

**Review by JanAckermann — APPROVED** [review]

File: src/components/OcTextarea.vue [hunk 1]
```diff
@@ -1,51 +1,148 @@
...
-    class="uk-textarea"
-    :value="value"
-    :placeholder="placeholder"
-    :aria-label="label"
-    @input="$_ocTextArea_onInput($event.target.value)"
-    @focus="$_ocTextArea_onFocus($event.target.value)"
-    @keydown="$_ocTextArea_onKeyDown($event)"
-  />
+  <div>
+    <label :id="labelId" class="oc-label" :for="textareaId" v-text="label" />
```
Comment (JanAckermann) [inline src/components/OcTextarea.vue:3]: do we still need the label id?

Comment (kulmann) [reply src/components/OcTextarea.vue:3]: Not needed anymore

File: src/components/OcTextarea.vue [hunk 2]
```diff
@@ -1,51 +1,145 @@
...
   name: "OcTextarea",
   status: "review",
   release: "1.0.0",
   props: {
     /**
-     * Text value of the form textarea.
+     * The ID of the element.
      */
-    value: {
+    id: {
```
Comment (JanAckermann) [inline src/components/OcTextarea.vue:56]: We have already an id here, replace the default with unique-id and rename old textareaId :+1:

**Review by JanAckermann — CHANGES_REQUESTED** [review]

**Review by JanAckermann — APPROVED** [review]

**Review by kulmann — CHANGES_REQUESTED** [review]
Could you introduce the new styles as a style tag within the `OcTextarea.vue` file (i.e. as a real single file component) instead of in a new scss file? You can do so by adding a `<style lang="scss"></style>` tag as third element (order is: `template`, `script`, `style`, `docs`).

**Review by kulmann — APPROVED** [review]

**Pull request merged**